assets/js/components/Work/Tabs/FileSetModal.jsx

```diff
@@ -0,0 +1,309 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { Button } from "@nulib/admin-react-components";
+import { useDropzone } from "react-dropzone";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Error from "@js/components/UI/Error";
+import { GET_PRESIGNED_URL } from "@js/components/IngestSheet/ingestSheet.gql.js";
+import { GET_WORK, INGEST_FILE_SET } from "@js/components/Work/work.gql.js";
+import { useQuery, useMutation } from "@apollo/client";
+import { s3Location, toastWrapper } from "@js/services/helpers";
+import { useForm, FormProvider } from "react-hook-form";
+import UIFormInput from "@js/components/UI/Form/Input.jsx";
+import UIFormField from "@js/components/UI/Form/Field.jsx";
+import UIFormSelect from "@js/components/UI/Form/Select.jsx";
+import { FILE_SET_ROLES } from "@js/services/global-vars";
+import UISkeleton from "@js/components/UI/Skeleton";
+
+/** @jsx jsx */
+import { css, jsx } from "@emotion/core";
+const dropZone = css`
+  background: #efefef;
+  border: 3px dashed #ccc;
+`;
```

Comment:
We should probably globalize these styles into `.scss` now that we're re-using the React Dropzone component.